src/pages/Search/Search.module.scss

```
@@ -0,0 +1,168 @@
+@use "src/utils/styles/mediaQuery" as media;
+
+.search {
+  width: 100%;
+  height: 100vh;
+  font-size: 32px;
+
+  @include media.media-breakpoint-up(mobile) {
+    max-width: 720px;
+  }
+
+  // 상단 navbar
+  &__nav {
+    display: flex;
+    margin-left: 30px;
+    margin-top: 72px;
+    align-items: center;
+
+    &__text {
+      font-size: 18px;
+      font-weight: 500;
+      width: 32px;
+      height: 22px;
+      padding-left: 16px;
+    }
+
+    &__button {
+      &--previous-button {
+        cursor: pointer;
+        background: none;
+        border: none;
+        width: 20px;
+        height: 20px;
+      }
+    }
+
+    &__icon {
+      &__previous-icon {
+        width: 7px;
+        height: 14px;
+      }
+    }
+  }
+
+  // 검색 추천 문구
+  &__recommend {
+    height: 80px;
+    margin-top: 64px;
+    margin-right: 60px;
+    margin-left: 30px;
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 40px;
+    white-space: pre-wrap;
+    color: #000000;
+  }
+
+  // 검색창
+  &__bar {
+    display: flex;
+    border-radius: 15px;
+    border: none;
+    height: 30px;
+    align-items: center;
+    margin-top: 32px;
+    margin-right: 16px;
+    margin-left: 16px;
+    background: #eeeeee;
+
+    &__input {
+      width: 100%;
+      height: 15px;
+      border: none;
+      background: none;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 15px;
+      margin-left: 11px;
+
+      &::placeholder {
+        color: #c4c4c4;
+        font-size: 12px;
+      }
+    }
+
+    &__input:focus {
+      outline: none;
+    }
+
+    &__icon {
+      width: 18px;
+      height: 18px;
+      padding-right: 14px;
+      cursor: pointer;
+    }
+  }
+
+  // 트렌드 롤링 배너
+  &__trend-banner {
+    display: flex;
+    overflow: hidden;
+    margin-top: 8px;
+    margin-right: 27px;
+    margin-left: 27px;
+    height: 15px;
+    font-size: 12px;
+
+    strong {
+      padding-right: 16px;
+    }
+
+    &__tag-list {
+      display: flex;
+      flex: 0 0 auto;
+      white-space: nowrap;
+      overflow: hidden;
+      list-style: none;
+
+      &__tag {
+        animation: text-loop 10s linear infinite;
+        font-weight: 400;
+        padding-right: 10px;
+      }
+
+      @keyframes text-loop {
+        0% {
+          transform: translate3d(0, 0, 0);
+        }
+
+        100% {
+          transform: translate3d(-100%, 0, 0);
+        }
+      }
+    }
+  }
+
+  // 검색창 결과 리스트
+  &__query-list {
+    margin-top: 24px;
+
+    &__item {
+      display: flex;
+      font-size: 14px;
+      height: 34px;
+      padding-left: 52px;
+      align-items: center;
+      cursor: pointer;
+
+      &:hover {
+        background: #eeeeee;
+      }
+    }
+
+    &__not-found {
```

```suggestion
    &__not-found {
      work-break: keep-all;
```
해당 속성을 추가하면 보기에 더 좋을 것 같습니다.
> 관련 음
> 식점 

같은 상황을 피할 수 있습니다.

---

이런 속성이 있었네요.....😮 배워갑니다!